src/static/js/client/sidebar-search.js

```diff
@@ -325,6 +342,16 @@ export function addPageListeners() {
       }, settings.stoppedTypingDelay);
   });
 
+  info.searchInput.addEventListener('keydown', domEvent => {
+    if (domEvent.key === 'ArrowDown') {
+      const elem = info.results?.firstChild;
```

Comment:
You don't need to do a non-nullish check on `info.results` here. It's declared [as part of](https://github.com/hsmusic/hsmusic-wiki/blob/15bc0f97e630a1b1d13d0bd80096d81b9a65d7de/src/static/js/client/sidebar-search.js#L264-L265) `mutatePageContent`, which always happens [before](https://github.com/hsmusic/hsmusic-wiki/blob/15bc0f97e630a1b1d13d0bd80096d81b9a65d7de/src/static/js/client/index.js#L42-L48) the step you're modifying (`addPageListeners`); and no code sets it back to null.

```suggestion
      const elem = info.results.firstChild;
```